catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/UpdateOperations.java

```diff
@@ -205,8 +206,10 @@ public UpdateResponse update(UpdateStorageRequest streamUpdateRequest)
                             true,
                             true);
 
-                    updatedMetacard.setAttribute(new AttributeImpl(Metacard.RESOURCE_SIZE,
-                            String.valueOf(contentItem.getSize())));
+                    if (StringUtils.isBlank(contentItem.getQualifier())) {
```

Comment:
Maybe this if statement should go at the beginning of the for loop like in https://github.com/codice/ddf/blob/master/catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/CreateOperations.java#L656.